src/index.js

```diff
@@ -265,9 +277,14 @@ const extendCodeBlockWithSyntaxHighlighting = (settings) => {
 					</InspectorControls>
 					{(() => {
 						if (useBlockProps) {
+							const blockProps = useBlockProps();
+
+							// Copy the styles to ensure that the code-block-overlay is updated when the font size is changed in Gutenberg 9.5+.
+							textAreaProps.style = blockProps.style;
```

Comment:
*This* is what I was missing!! I was exploring to see how Gutenberg had changed but didn't think to copy over the styles. I love it :+1: